### ISSUE TITLE
WebShare Proxy

### ISSUE THREAD

**Issue by juliomoreirx** [gui-fkb/twitch-accounts#6]
How to use my webshare proxies in rotate mode?

**Comment by gui-fkb** [comment]
Hey! Try to use the following pattern: http://user:pass@ip:port inside config.go file

don't forget to use the http prefix

**Comment by juliomoreirx** [comment]
Thanks dud, Solved. Can i help me on this problem now, please

How many accounts do you want to create?
1
Creating account 1 of 1
Session is alive.
Email address is l.o.velas.c.a.n.tte@gmail.com.
Getting twitch cookies.
Getting kasada code
Getting local integrity token
IntegrityToken: v4.local.YwUazqG_v0uJ5zzZxsb5PFUFmij55fYLbyzHJeC+313...
Creating account...
Parsed error response: {Breached:false Error:Oops! We encountered an unexpected error. Please try again. Errors:[Oops! We encountered an unexpected error. Please try again.] ErrorCode:5025}
Finished accounts creation.

**Comment by gui-fkb** [comment]
If i'm sure, i remember getting this error because of something related to bad proxy/bad ip error. Maybe try again more times or try a different proxy provider, I recommend using ProxyLogic.

If your local IP is not blocked by Twitch you can also try to run the bot without proxy, just let the proxy string empty on config.go

**Comment by juliomoreirx** [comment]
Alright! Thanks for trying to help me. Maybe one of my proxies was blocked. Without a proxy I tested and managed to create two accounts and then I think it was blocked. I will test other proxies and create them little by little. Thanks again!

**Comment by gui-fkb** [comment]
Yeah, you can assure if your IP is really blocked by going manually in twitch via your browser and trying to create an account. If  you can't manage to create, it means you have been blocked.

**Comment by gui-fkb** [comment]
I believe it has been resolved for now. If you have any further questions or concerns, feel free to open a new issue or contact me through discord. I would be glad to assist you
